### PR TITLE
Minor design improvements

### LINKED ISSUE #36: Fix spacing in HTML
Adding some HTML line breaks and CSS padding for spacing.

### PR DESCRIPTION
**Pull request by mitchfeldman**
1. Bolded `.navbar-brand`
2. Fixed spacing issues on `assignment_detail.html`
3. Added bottom margins to headers
4. Changed verbiage edit students button in `course_detail.html` to "Add/Remove Students" (was Add/Remove Students / Edit Course)
5. Fixed button column on `environment_definition_list.html`
6. Made "Create buttons" uniform on `assignments_list.html`, `coures_detail.html`, `environment_definition_list.html`

Fixes #36

### CODE REVIEW

File: cvcl/templates/assignment_create.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
 {% block content %}
     <h1>Create Assignment</h1>
 
+    <br>
```
Comment (DSpeichert) [inline cvcl/templates/assignment_create.html]: That's not spacing, that's new line. You should add a margin on h1 instead, in CSS.

**Review by DSpeichert — CHANGES_REQUESTED** [review]

**Review by DSpeichert — CHANGES_REQUESTED** [review]
CSS should be in a CSS file, not inline.

File: cvcl/templates/master.html [hunk 2]
```diff
@@ -5,7 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    {% block title %}<title>Cloud VCL</title>{% endblock %}
```
Comment (DSpeichert) [inline cvcl/templates/master.html]: Why did you remove that?

File: cvcl/templates/master.html [hunk 3]
```diff
@@ -29,7 +28,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="{% url 'index' %}">Cloud VCL</a>
+            <a class="navbar-brand" href="{% url 'index' %}" style="padding-top:15px;">Cloud VCL</a> <!-- Padding is necessary with material theme -->
```
Comment (DSpeichert) [inline cvcl/templates/master.html]: To CSS.

**Review by DSpeichert — CHANGES_REQUESTED** [review]

File: cvcl/templates/master.html [hunk 4]
```diff
@@ -13,6 +15,7 @@
     <!-- Add additional CSS in static file -->
     {% load static %}
     <link rel="stylesheet" href="{% static "style.css" %}">
+
```
Comment (DSpeichert) [inline cvcl/templates/master.html]: why?

File: cvcl/templates/master.html [hunk 5]
```diff
@@ -28,7 +31,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="{% url 'index' %}" style="padding-top:15px;">Cloud VCL</a> <!-- Padding is necessary with material theme -->
+            <a id="navbrand" class="navbar-brand" href="{% url 'index' %}">Cloud VCL</a> <!-- Padding is necessary with material theme -->
```
Comment (DSpeichert) [inline cvcl/templates/master.html]: Is the navbar-brand class used anywhere else? Why not use it to identify the style?

**Review by DSpeichert — CHANGES_REQUESTED** [review]

**Review by DSpeichert — APPROVED** [review]

**Comment by abhishekvalanju** [comment]
Redo please, the Assignment Details/Update pages look horrible as shown by the screenshot I provided. Also restore that column you deleted from Courses detail and Environment Definition Details

File: cvcl/templates/environment_definition_list.html [hunk 6]
```diff
@@ -14,16 +14,16 @@
     <table class="table table-hover">
         <tr>
             <th>Name</th>
-            <th>Actions</th>
```
Comment (abhishekvalanju) [inline cvcl/templates/environment_definition_list.html:17]: Add this back in

File: cvcl/templates/environment_definition_list.html [hunk 7]
```diff
@@ -14,16 +14,16 @@
...
         </tr>
         {% for ed in environmentdefinition_list %}
             <tr>
-                <td><a href="{% url "envdefs.detail" ed.id %}">{{ ed.name }}</a></td>
-                <td>
```
Comment (abhishekvalanju) [inline cvcl/templates/environment_definition_list.html:22]: Add the <td> back

**Comment by mitchfeldman** [comment]
Assignments_detail is out of scope for this but you're right, I overlooked it. I can create a separate issue for that.

If you can show me what you're seeing on your end with the columns that'd be great.

File: cvcl/static/style.css [hunk 8]
```diff
@@ -1,3 +1,12 @@
...
     padding-top: 70px;
-}
\ No newline at end of file
+}
+
+h1 {
+    margin-bottom: 25px;
+}
+
+#navbrand {
```
Comment (DSpeichert) [inline cvcl/static/style.css]: Delete this, it doesn't do anything.

File: cvcl/templates/assignment_create.html [hunk 9]
```diff
@@ -13,9 +13,9 @@
...
                 <a href="{% url "assignments" %}" class="btn btn-default">
-                    {% bootstrap_icon "remove" %} Back
+                    {% bootstrap_icon "chevron-left" %} Back
                 </a>
             {% endbuttons %}
         </div>
     </form>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```
Comment (DSpeichert) [inline cvcl/templates/assignment_create.html:21]: Add new line at the end of file

File: cvcl/templates/index.html [hunk 10]
```diff
@@ -39,10 +39,10 @@
         </p>
 
         <ol>
-            <li>Create course from Courses</li>
-            <li>Create environment definition from Environment Definitions</li>
-            <li>Create VM definition within Environment Definition</li>
-            <li>Create assignment from Assignments</li>
+            <li>Create course from <a href="{% url "courses" %}">Courses</a></li>
```
Comment (DSpeichert) [inline cvcl/templates/index.html]: This is inline help text. Remove it from here. It shouldn't be visible to students either. Add this back where appropriate according to #40.

File: cvcl/templates/master.html [hunk 11]
```diff
@@ -37,7 +37,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="{% url 'index' %}">Cloud VCL</a>
+            <a id="navbrand" class="navbar-brand" href="{% url 'index' %}">Cloud VCL</a> <!-- Padding is necessary with material theme -->
```
Comment (DSpeichert) [inline cvcl/templates/master.html]: Undo, not necessary.

**Review by DSpeichert — CHANGES_REQUESTED** [review]

**Review by DSpeichert — APPROVED** [review]

**Review by abhishekvalanju — APPROVED** [review]
Looks good

**Pull request merged**